### DISCUSSION TITLE
Search for channels?

### DISCUSSION THREAD

**Discussion by abceleung** [twitch4j/twitch4j#918] [Q&A] [answered]
Hi, I want to use the Twitch API to search for channels:
https://dev.twitch.tv/docs/api/reference/#search-channels
I am unable to find this feature in the Twitch4j docs. Is this feature not implemented? 
Thanks

**Comment by iProdigy** [comment, marked as answer]
That endpoint is implemented as `TwitchHelix#searchChannels` - I would recommend using the javadocs for more updated documentation than our docusaurus page: https://twitch4j.github.io/javadoc/com/github/twitch4j/helix/TwitchHelix.html#searchChannels(java.lang.String,java.lang.String,java.lang.Integer,java.lang.String,java.lang.Boolean)
